Formula/unnethack.rb

```diff
@@ -20,10 +20,6 @@ class Unnethack < Formula
   option "with-curses-graphics", "Enable curses graphics (play with fanciness)"
 
   def install
-    # crashes when using clang and gsl with optimizations
-    # https://github.com/mxcl/homebrew/pull/8035#issuecomment-3923558
-    ENV.no_optimization
```

Comment:
should this be ENV.O0?

Reply:
Yeh, it could be. Given it was a no-op and no-one seems to have complained I'm tempted to just remove it as it seems unlikely it's needed.

Reply:
ok